lib/index.js

```diff
@@ -257,6 +272,88 @@ Intercom.prototype.createNewJob = function(msg, key, fn){
 };
 
 /**
+ * Identify a user in intercom
+ *
+ * https://doc.intercom.io/api/#create-or-update-user
+ *
+ * @param {Identify} identify
+ * @param {Function} fn
+ * @api public
+ */
+
+Intercom.prototype.identifyV1 = function(identify, fn){
```

Comment:
That's certainly the cleaner way but I tried that first and ran into a whole bunch of dependency issues and what not so as to mitigate the SEV asap (and given that this is all temp) I figured just do the quick and dirty c&p.